templates/osg_csv.sh.in

```diff
@@ -1,6 +1,8 @@
 export COPYRECO=%COPYRECO%
 export COPYFULL=%COPYFULL%
 export COPYLOG=%COPYLOG%
+export USERUCIO=%USERUCIO%
+export X509_USER_PROXY=%X509_USER_PROXY%
```

Comment:
This environment script gets executed in the payload. I am trying to avoid the scenario where someone sets X509_USER_PROXY=local_dir/x509_user_proxy and i wasn't sure if condor would preserve the local directory structure when transferring files using transfer_input_files https://htcondor.readthedocs.io/en/latest/users-manual/file-transfer.html